defaults/main.yml

```diff
@@ -30,3 +30,9 @@ thinlinc_tlsetup_migrate_conf: "old"
 #
 # The default value is null, which is equal to ansible_fqdn.
 thinlinc_agent_hostname: null
+
+# Define if this is a standalone thinlinc environment or a cluster
+thinlinc_use_cluster: false
```

Comment:
What does your scenario look like here? How is this different from just having a single agent specified?

Reply:
Hi Pierre,

I am using thinlinc on a single server only. When i tried to run the playbook for the first time on multiple servers (i have 2 servers running thinlinc) the hostnames of the servers were added to the two config files. Because the host is a master and a agent.

Reply:
Ah, I see. In that case I think your suggestion here is more of a workaround than a fix. If you try to deploy two multi-machine clusters then things would still break.

So I think we need to look at a proper fix for #1. :/